homeassistant/components/enigma2/media_player.py

```diff
@@ -253,17 +253,13 @@ def device_state_attributes(self):
         currservice_begin:  is in the format '21:00'.
         currservice_end:    is in the format '21:00'.
         """
-        attributes = {}
         if not self.e2_box.in_standby:
```

Comment:
I know it's not for this PR, but the code would look more beautiful if we turn it around: `if standby: return {}`, giving more line length for the actual code.

Reply:
That is a great idea, changed in https://github.com/home-assistant/core/pull/41271/commits/ecbd82112131ad2bb765f62604b769f1a6ccd227

Reply:
This ofc is not just in this case, but in many of these cases in this PR. Anyway, not really for this PR, feel free to merge without.

Reply:
Oh gotcha. Will revert my last commit and open a separate PR to clean up these cases.